src/main.js

```diff
@@ -716,13 +707,8 @@ class MiniGraphCard extends LitElement {
   }
 
   numberFormat(num, language, dec) {
-    if (!Number.isNaN(Number(num)) && Intl) {
-      if (dec === undefined || Number.isNaN(dec)) {
```

Comment:
I don't remember exactly why I added this guard, but I suspect there was issues if `dec` was `undefined` or not a number when used in `Intl.NumberFormat()` below.

Would be great if you could double check this.

Reply:
I double-checked it before committing, it was added in https://github.com/kalkih/mini-graph-card/commit/875b55743a0152e7630a6111d7128a2a28657c97 (not by you :) )
if it's undefined, it works as intended (undefined is the default after all), if it's NaN it's already failing earlier (thus no functional difference, and config should be considered invalid in that case regardless => should be checked at that level with the upcoming milestone)